python/ray/services.py

```diff
@@ -409,11 +415,22 @@ def start_ray_processes(address_info=None,
       start a global scheduler process.
     include_redis (bool): If include_redis is True, then start a Redis server
       process.
+    num_cpus: A list of length num_local_schedulers containing the number of
```

Comment:
We're probably going to need to pass in more resources in the future. It may be a good idea to use a Python namedtuple to define a `ResourceList` type, so that we can pass in a single argument for all the resources, and not have to remember the ordering of such an argument.